types/checkout/checkout.d.ts

```diff
@@ -88,20 +88,33 @@ interface CheckoutOpenOptionsWithDiscountCode {
   discountId?: never;
 }
 
+interface CheckoutOpenOptionsWithCustomer {
+  customer?: CheckoutCustomer;
+  customerAuthToken?: never;
+  savedPaymentMethodId?: string;
```

Comment:
I wonder if `savedPaymentMethodId` does actually require the auth token to be present

```suggestion
  savedPaymentMethodId?: never;
```

Reply:
It won't break if it is provided without a `customerAuthToken` it just won't do anything, maybe best to add the restriction